compiler/modules/nback/build.bal

```diff
@@ -2129,7 +2136,7 @@ function semTypeRepr(t:SemType ty) returns Repr|BuildError {
         TaggedRepr repr = { base: BASE_REPR_TAGGED, llvm: LLVM_TAGGED_PTR, subtype: w };
         return repr;
     }
-    return err:unimplemented("unimplemented type (" + w.toHexString() + ")");
+    return err:unimplemented("unimplemented type (" + w.toHexString() + ")", loc);
```

Comment:
I think the easiest thing to do is to turn this into a panic.  I don't think we're relying on this anywhere. See if any of our test cases execute this line.

Reply:
changed to panic